Mathlib/GroupTheory/Subgroup/Basic.lean

```diff
@@ -2646,6 +2646,10 @@ theorem rangeRestrict_surjective (f : G →* N) : Function.Surjective f.rangeRes
 #align monoid_hom.range_restrict_surjective MonoidHom.rangeRestrict_surjective
 #align add_monoid_hom.range_restrict_surjective AddMonoidHom.rangeRestrict_surjective
 
+@[to_additive (attr := simp)]
+lemma rangeRestrict_injective_iff {f : G →* N} : Injective f.rangeRestrict ↔ Injective f :=
+  forall₂_congr $ fun x y ↦ by simp [Subtype.ext_iff]
```

Comment:
Not tested:
```suggestion
  Set.injective_codRestrict _
```

Reply:
This works provided I also use `convert`.